Helper/Api.php

```diff
@@ -291,8 +297,8 @@ public function buildRequest($requestData)
         $requestMethod = empty($requestData->getRequestMethod()) ? 'POST' : $requestData->getRequestMethod();
         $apiUrl        = $this->getFullApiUrl($dynamicApiUrl);
 
-        $headers    = (array)$requestData->getHeaders();
-        $statusOnly = (bool)$requestData->getStatusOnly();
+        $headers    = (array) $requestData->getHeaders();
```

Comment:
mostly just curious: is this some PHPCS thing? whats the official guideline on these spaces? @daisy1754 or @matt-thomason

Reply:
This isn't explicitly covered by PSR-2, and that's why our codesniffer doesn't care either way here. Personally I like the spaces, as it better matches what we expect in control structures with parens/braces, and with things like closures.
https://www.php-fig.org/psr/psr-2/

Reply:
Though since it's the linter making changes, perhaps it's an addition to MEQP-2 or I'm misreading the spec. Either way, I like this 👍